src/prism.c

```diff
@@ -7106,7 +7106,8 @@ parser_lex(pm_parser_t *parser) {
                     else if(
                         lex_state_beg_p(parser) ||
                         (lex_state_p(parser, PM_LEX_STATE_FITEM) && (peek(parser) == 's')) ||
-                        lex_state_spcarg_p(parser, space_seen)
+                        lex_state_spcarg_p(parser, space_seen) ||
+                        parser->in_keyword_arg
```

Comment:
I have the same issue here that I have in the other one in that this doesn't match up to `parse.y`. I think if we fix the other one this one will probably be fixed as well.